src/metric_config.cpp

```diff
@@ -36,7 +36,7 @@ bool MetricConfig::validateEndpointPath(const std::string& endpoint) {
 }
 
 // Getting the "monitoring" metrics config as input
-Status MetricConfig::parseMetricsConfig(const rapidjson::Value& metrics, uint64_t restPort) {
+Status MetricConfig::parseMetricsConfig(const rapidjson::Value& metrics, bool failIfMetricsAreEnabled) {
```

Comment:
```suggestion
Status MetricConfig::parseMetricsConfig(const rapidjson::Value& metrics, bool forceFailureIfMetricEnabled) {
```
Also add comment why it is done this way

Reply:
Done.